tests/tests_integration/conftest.py

```diff
@@ -103,3 +103,23 @@ def make_cognite_client(beta: bool = False) -> CogniteClient:
             **beta_configuration,
         )
     )
+
+
+@pytest.fixture(scope="session")
+def cognite_client_cdf_authenticated() -> CogniteClient:
```

Comment:
`cognite_client`, defined in the fixture above is already an authenticated `CogniteClient` - what is the specific need for a different/extra one? 

If we need client credentials in particular (if yes, why?), I suggest you rename this fixture and explain why in a comment.